### DISCUSSION TITLE
Bug on results page 2

### DISCUSSION THREAD

**Discussion by MatiasERP** [pierreaubert/spinorama#279] [General]
When I search for a keyword say "Revel" the 1st page of the results is ok:
https://www.spinorama.org/?page=1&search=revel

But on the bottom, either number 2 or Next buttons both lead me to home page results page 2, and not Revel results page.
https://www.spinorama.org/?page=2

It just needs the search term suffix on their URLs like:
https://www.spinorama.org/?page=2&search=revel

**Comment by pierreaubert** [comment]
Hum I do not see the same thing. It is possible that I didnt cleanup the
cache of old versions. Can you cleanup your cache and tell me if you see
the same thing?

Pierre

On Thu, Jun 27, 2024 at 6:19 PM MatiasERP ***@***.***> wrote:

> When I search for a keyword say "Revel" the 1st page of the results is ok:
> https://www.spinorama.org/?page=1&search=revel
>
> But on the bottom, either number 2 or Next buttons both lead me to home
> page results page 2, and not Revel results page.
> https://www.spinorama.org/?page=2
>
> It just needs the search term suffix on their URLs like:
> https://www.spinorama.org/?page=2&search=revel
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/279>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH2BIL52CU45R47KBWDZJQ3SBAVCNFSM6AAAAABKAIIU52VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHA3TANBVGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
--
PfA

**Comment by MatiasERP** [comment]
I just tried with another browser for clean cache (Edge) and same results.

I enter spinorama.org. Then I search for Revel. Go to bottom and the next buttons are for home page 2 results.

I think the search box updates the URL and the results in real time, which is nice, but the buttons below are not being updated, they are kept from the former page.

**Comment by pierreaubert** [comment]
definitively broken, I will debug.

On Fri, Jun 28, 2024 at 4:04 PM MatiasERP ***@***.***> wrote:

> I just tried with another browser for clean cach (Edge) and same results.
>
> I enter spinorama.org. Then I search for Revel. Go to bottom and the next
> buttons are for home page 2 results.
>
> I think the search box updates the URL and the results in real time, which
> is nice, but the buttons below are not being updated, they are kept from
> the former page.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/279#discussioncomment-9905478>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH4G4VKIMOBRNPLZMUDZJVUQRAVCNFSM6AAAAABKAIIU52VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMBVGQ3TQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
--
PfA

**Comment by pierreaubert** [comment]
Should be fixed with https://github.com/pierreaubert/spinorama/commit/3b313f50f2ed693515a3ca62d7c2f878708cd649

**Comment by MatiasERP** [comment]
I did CTRL-F5 to get the newest version. Now on home page, if I search for KEF for example, there are no Next Page and page number buttons. I thought it was fixed, but then I noticed the Blades were missing, so I search for them and they showed up. So I guess instead of showing filtered next page and page number buttons, it just shows 1st page results and nothing more?
